### PR TITLE
Includes branding doc in replication map

### PR DESCRIPTION
**Pull request by garethbowen**
# Description

medic/medic-webapp#5027

# Review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://github.com/medic/medic-docs/blob/master/development/style-guide.md), documented if necessary.
- [ ] Documented: Announced in Changes.md in plain English. Configuration and user documentation on [medic-docs](https://github.com/medic/medic-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in Changes.md.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: ddocs/medic/views/docs_by_replication_key/map.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 function (doc) {
   if (doc._id === 'resources' ||
+      doc._id === 'branding' ||
```
Comment (garethbowen) [inline ddocs/medic/views/docs_by_replication_key/map.js]: This is the important bit - this view is how we work out how to replicate docs to offline users. Adding this makes sure all users can get it.

File: tests/e2e/api/controllers/_changes.spec.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ const DEFAULT_EXPECTED = [
   'appcache',
   'settings',
   'resources',
+  'branding',
```
Comment (garethbowen) [inline tests/e2e/api/controllers/_changes.spec.js]: This adds a test to ensure all users get the branding doc.

File: webapp/src/js/services/branding-images.js [hunk 3]
```diff
@@ -72,26 +70,19 @@ angular.module('inboxServices').factory('BrandingImages',
...
-          DB().get(BRANDING_ID).then(doc => {
-            resolve(doc.title);
-          }).catch(err => {
-            reject(err);
-          });
-        });
-      },
-      replacePlaceholders: function($elem) {
-        init.then(function() {
+      getAppTitle: () => DB().get(BRANDING_ID).then(doc => doc.title),
```
Comment (garethbowen) [inline webapp/src/js/services/branding-images.js]: You don't need to make a `new Promise` if you already have a promise, so this function becomes a lot cleaner and a little faster too.

The rest of the changes in this file are just ES6 style code for readability.

File: ddocs/medic/views/docs_by_replication_key/map.js (hunk 1 above)
Comment (Jodge) [reply ddocs/medic/views/docs_by_replication_key/map.js]: I see

File: tests/e2e/api/controllers/_changes.spec.js (hunk 2 above)
Comment (Jodge) [reply tests/e2e/api/controllers/_changes.spec.js]: I see

File: webapp/src/js/services/branding-images.js (hunk 3 above)
Comment (Jodge) [reply webapp/src/js/services/branding-images.js]: Nice!

**Pull request merged**